AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -662,6 +845,23 @@ - (void)moveRowAtIndexPath:(NSIndexPath *)indexPath toIndexPath:(NSIndexPath *)n
   }];
 }
 
+#pragma mark - Data Querying (Subclass API)
+
+- (NSArray *)indexPathsForEditingNodesOfKind:(NSString *)kind
+{
+  return _editingNodes[kind] != nil ? ASIndexPathsForMultidimensionalArray(_editingNodes[kind]) : [NSArray array];
```

Comment:
Do we need the [NSArray array]?  Using ASIndexPathsForMultidimensionalArray is pretty inefficient, so I am concerned we might want to eliminate this convenience or audit its usage carefully so the cost is not hidden.